packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -964,37 +903,108 @@ describe("Garbage Collection Tests", () => {
             );
         });
 
-        describe("SweepReady usage error enabled (main container)", () => {
-            const snapshotCacheExpiryMs = 500;
-            const sweepTimeoutMs = defaultSessionExpiryDurationMs + snapshotCacheExpiryMs + oneDayMs;
+        describe("Main Container Behavior", () => {
```

Comment:
Not sure how I feel about the term "main container". I feel like this is adding yet another term to the ones already there :-). For instance, in the summarizer world, the terms are "summarizer client" and "parent client" that spawns the "summarizer client". All other clients are just "regular clients".

Reply:
Oh ok good to know.  I can have a follow-up PR to clean up the naming.  "Parent Client" is better?

Reply:
Renamed to interactive client